build_debian.sh

```diff
@@ -395,6 +395,12 @@ sudo LANG=C DEBIAN_FRONTEND=noninteractive chroot $FILESYSTEM_ROOT apt-get -y in
 	lsof                    \
 	sysstat
 
+# latest tcpdump control resource access with AppArmor.
+# override tcpdump profile to allow tcpdump access TACACS config file.
+if [ -f ${FILESYSTEM_ROOT}/etc/apparmor.d/local/usr.bin.tcpdump ]; then
+    sudo LANG=C chroot $FILESYSTEM_ROOT /bin/bash -c "echo -e '\n# for access tacacs config\n/etc/tacplus_nss.conf r,\n' > /etc/apparmor.d/local/usr.bin.tcpdump"
```

Comment:
there is local file why to add some files in the buildimage, this does not seem to be the standard way.

Reply:
This code is modifying the local file: '/etc/apparmor.d/local/usr.bin.tcpdump'
The local file been created by tcpdump install.

Reply:
@lguohan , Could you please check this PR again? I reply your comments, current code does not add new file in buildimage, when install tcpdump, the package will install a local apparmer profile, and current code will check and modify the local apparmer profile.

Reply:
the local apparmer profile is empty, right?

it is better to make the local file more deterministic 

suggest to use this approach.

sudo cp files/dhcp/rfc3442-classless-routes $FILESYSTEM_ROOT/etc/dhcp/dhclient-exit-hooks.d

Reply:
@lguohan , thanks, fixed according to your suggestion, please check.